### ISSUE TITLE
vmcloak dependencies install issue

### ISSUE THREAD

**Issue by geeksailor** [hatching/vmcloak#215]
Good afternoon devs,
I am attempting to create a guest VM using VMcloak that I intend to use with Cuckoo3 following the steps provided in the documents for the install process. However in following the steps, every time it seems like I get up to the dependencies install stage and it freezes at  
> applying settings to optimize IE11

 part of the install. I have tried this on multiple systems and I always seem to get to this stage then it hangs. In the past I have ended up waiting 4-6 hours in the past before halting it but it always seems to freeze here. I decided to make a guest VM once again last night using my bulky gaming pc. I created an Ubutu VM and  gave it considerable more resources since past VM hosts I have tried to do this on before were a bit smaller and had less processing power and memory. I thought that might be the issue with it taking so long and this might fix it. I got to the dependencies install step of the install, let it run, and left it alone to run overnight, giving it more time to run this time. It is now going on around 18 hours of the install and it still is sitting on this step of the install with no error outputs and nothing seeming to happen. Is there anything you can think of I can try to see what may be happening on the guest VM or if you know if any known bugs with the install process at this time?
Thank you!

P.S. I'm not sure who created the info to connect this project to projects page on readthedocs.io but that seems to be quite out of date compared to the most recent docs provided in the build. I guess it reads a yaml from your git for what is displayed on their site? Not sure if that is something you want to try and run down and update to provide a more up to date install process online.

**Comment by krapgras** [comment]
Hey Geeksailor,

Check this issue:

https://github.com/hatching/vmcloak/issues/197

That explains what is happening, however this repo isn't maintained at all by the authors.
Additionally it was made for cuckoo v2 and not v3. I got some changes to work with python3 but it is far from fully compatible with cuckoo v3.

I'm sure you can make a lot of changes yourself but it will definitely be a adventure!

**Comment by Cryss76** [comment]
I made a fix and pull request for this issue which where never added. So eventually closed the request. But feel free to try my fork.

> Additionally it was made for cuckoo v2 and not v3.

True and not at the same time. Yeah it was made for cuckoo 2 but it was changed rather hastely to somewhat work with cuckoo3. So it would work for cuckoo3 if there weren't any bugs but it wont work for cuckoo2 at all  (at least not unmodified).

**Comment by geeksailor** [comment]
Oh awesome! Thank you all for the quick response! @Cryss76,I'll go check that out right now!
Thanks again!
Cheers!

**Comment by geeksailor** [comment]
@Cryss76 I found the updates you made to the python scripts on your fork of the project and I made those updates to my own  scripts. That work around did seem to work mostly and I managed to get some of the dependencies that were hanging up before to work. But I am still experiencing hang-ups on pillow and office for now. I guess the async work around may not still fully work with those dependencies? I'll keep testing but for now it may be something I may have to go without. 

I'll close this ticket out since #197 is still open and hopefully one day this gets fully resolved. Thank you again for your assistance!
Cheers!
